app/controllers/course/material/folders_controller.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 class Course::Material::FoldersController < Course::Material::Controller
   def show
+    @subfolders = @folder.children.with_content_statistics.accessible_by(current_ability).
+                  includes(:owner).without_empty_linked_folder
```

Comment:
@lowjoel I don't remember why I used `preload` before, seems there were issues with edge, but now `includes`  also works and no tests break.